### PR TITLE
Bug 1532693 -  Use lightbulb icon for feature recommendation message

### PR DESCRIPTION
**Pull request by piatra**
Phab changes https://phabricator.services.mozilla.com/D22331

image

### CODE REVIEW

File: lib/CFRPageActions.jsm [hunk 1]
```diff
@@ -61,10 +61,21 @@ class PageAction {
...
 
   async showAddressBarNotifier(recommendation, shouldExpand = false) {
+    const addonsRecommendationClassName = "cfr-addons-recommendation-icon";
+    const featuresRecommendationClassName = "cfr-features-recommendation-icon";
     this.container.hidden = false;
 
     this.label.value = await this.getStrings(recommendation.content.notification_text);
 
+    if (recommendation.category === "cfrAddons") {
+      this.button.classList.remove(featuresRecommendationClassName);
```
Comment (k88hudson) [inline lib/CFRPageActions.jsm]: Could you maybe do something like:
```css
#cfr-button[data-cfr-icon="feature"] {
  background: url('...');
}
```

```js
this.button.setAttribute("data-cfr-icon", CATEGORY_ICONS[recommendation.category] || "");
```

**Comment by piatra** [comment]
The Phab patch needs to land first so we don't run into unreferenced files failures.

**Comment by k88hudson** [comment]
This looks good but is needs to be combined with MC changes, which are blocked on the changes in `CFR-feature-branch`. I'm going to combine it with those changes so they can land

**Comment by k88hudson** [comment]
Moving the image over would make this mergable actually

**Comment by k88hudson** [comment]
Merged to CFR pinned tabs via https://github.com/mozilla/activity-stream/pull/4839/commits/4fc8f0f7678c352cc5e9eeb5a572cbdbc168e4cd (icon will be added to MC patch)

**Pull request closed without merging**